embulk-api/src/main/java/org/embulk/spi/PageReader.java

```diff
@@ -181,19 +182,57 @@ public Instant getTimestampInstant(final int columnIndex) {
     }
 
     /**
+     * Returns a JSON value at the specified column in the {@code msgpack-java} representation.
+     *
+     * @param column  the column to get the JSON value
+     * @return the JSON value in the {@code msgpack-java} representation
+     * @deprecated Use {@link #getJsonValue(int)} instead.
+     *
      * @since 0.8.0
      */
+    @Deprecated
     public Value getJson(Column column) {
         return this.delegate.getJson(column);
     }
 
     /**
+     * Returns a JSON value at the specified column in the {@code msgpack-java} representation.
+     *
+     * @param columnIndex  the index of the column to get the JSON value
+     * @return the JSON value in the {@code msgpack-java} representation
+     * @deprecated Use {@link #getJsonValue(int)} instead.
+     *
      * @since 0.8.0
      */
+    @Deprecated
     public Value getJson(int columnIndex) {
         return this.delegate.getJson(columnIndex);
     }
 
+    /**
+     * Returns a JSON value at the specified column.
+     *
+     * @param column  the column to get the JSON value
+     * @return the JSON value in the {@code msgpack-java} representation
```

Comment:
Does this description require `in the {@code msgpack-java} representation` part?
IIUC, `msgpack-java` will remove in future release. This method returns just `JsonValue`.
I'm afraid of missing remove this part when `msgpack-java` removes it.

Reply:
Exactly! Thanks for the good catch, too.